packages/webdriver-utils/src/providers/genericProvider.js

```diff
@@ -69,15 +69,27 @@ export default class GenericProvider {
     }
   }
 
+  async getScrollDetails() {
+    return await this.driver.executeScript({ script: 'return [parseInt(window.scrollX), parseInt(window.scrollY)];', args: [] });
+  }
+
+  async getInitialScrollFactor() {
```

Comment:
why are we calling it scroll factor ? factor is usually ratio right ? we are just storing initial scroll location

Reply:
Done changed name to getInitialScrollLocation